dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/config/WSKeyStore.java

```diff
@@ -245,9 +247,13 @@ public WSKeyStore(String _name, Dictionary<String, Object> properties, KeystoreC
         }
 
         if ((type.equals(Constants.KEYSTORE_TYPE_JCERACFKS) || type.equals(Constants.KEYSTORE_TYPE_JCECCARACFKS)
-             || type.equals(Constants.KEYSTORE_TYPE_JCEHYBRIDRACFKS) || type.equals(Constants.KEYSTORE_TYPE_JAVACRYPTO)))
+             || type.equals(Constants.KEYSTORE_TYPE_JCEHYBRIDRACFKS) || type.equals(Constants.KEYSTORE_TYPE_JAVACRYPTO))) {
```

Comment:
Dumb question - I assume the 3 RACF types require the password to be "password", but does JAVACRYPTO (PKCS11) require that too?

Reply:
Good catch,  I've updated the code.

Reply:
Looks good!